src/context/TabContext.tsx

```diff
@@ -0,0 +1,26 @@
+import { ReactNode, createContext, useContext, useState } from 'react';
+
+type TabContextType = {
+  activeTab: string;
+  setActiveTab: (tab: string) => void;
```

Comment:
useState의 setter 함수를 타입으로 지정할 때는 이렇게 하는 것도 좋을 것 같아요 :)

```suggestion
  setActiveTab: Dispatch<SetStateAction<string>>;
```

Reply:
오호라~~~